python/notebooks/getting-started-gym.ipynb

```diff
@@ -0,0 +1,259 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Unity ML-Agents Toolkit\n",
+    "##  Gym Wrapper Basics\n",
+    "This notebook contains a walkthrough of the basic functions of the Python Gym Wrapper for the Unity ML-Agents toolkit. For instructions on building a Unity environment, see [here](https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Getting-Started-with-Balance-Ball.md)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 1. Load dependencies\n",
+    "\n",
+    "The following loads the necessary dependencies and checks the Python version (at runtime). ML-Agents Toolkit (v0.3 onwards) requires Python 3."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import sys\n",
+    "\n",
+    "from gym_unity.envs import UnityEnv\n",
+    "\n",
+    "%matplotlib inline\n",
+    "\n",
+    "print(\"Python version:\")\n",
+    "print(sys.version)\n",
+    "\n",
+    "# check Python version\n",
+    "if (sys.version_info[0] < 3):\n",
+    "    raise Exception(\"ERROR: ML-Agents Toolkit (v0.3 onwards) requires Python 3\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2. Start the environment\n",
+    "`UnityEnv` launches and begins communication with the environment when instantiated."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "env_name = \"../envs/GridWorld\"  # Name of the Unity environment binary to launch\n",
```

Comment:
Maybe here mention that for the "Gym Wrapper Basics", only environment with 1 agent is supported.